sunpy/net/scraper.py

```diff
@@ -26,13 +24,14 @@
 # added `%e` as for milliseconds `%f/1000`
 TIME_CONVERSIONS = {"{year:4d}": "%Y", "{year:2d}": "%y",
             "{month:2d}": "%m",
-            "{day:2d}": "%d",
-            "{hour:2d}": "%H",
+            "{month_name:w}": "%B",
+            "{month_name_abbr:3w}": "%b",
+            "{day:2d}": "%d", "{day_of_year:3d}": "%j",
+            "{hour:2d}": "%H", "{hour_12:2d}": "%I",
```

Comment:
I would be sorprised if any instrument/site uses a 12h format... and if so, this would need a way to input am/pm too.